targon/internal/targon/types.go

```diff
@@ -53,3 +53,14 @@ type GPUAttestationResponse struct {
 		SwitchID   string `json:"switch_id"`
 	} `json:"switch_claims,omitempty"`
 }
+
+type EmissionsData struct {
```

Comment:
this type should not exist. all the data we want is on the interval level which is what syncmongo syncs. just extend the struct syncmongo uses